cfme/tests/cli/test_rake_evm_automate-53.py

```diff
@@ -11,23 +11,8 @@
 pytestmark = [pytest.mark.ignore_stream("5.2")]
 
 
-@pytest.yield_fixture(scope="module")
-def back_up_default_domain(ssh_client):
-    ssh_client.run_command("rm -f /tmp/Default_backup.yaml")
-    rc = ssh_client.run_rake_command(
-        "evm:automate:export DOMAIN=Default YAML_FILE=/tmp/Default_backup.yaml "
-        "PREVIEW=false OVERWRITE=true")[0]
-    yield
-    if rc == 0:
-        rc, stdout = ssh_client.run_rake_command(
-            "evm:automate:import DOMAIN=Default YAML_FILE=/tmp/Default_backup.yaml PREVIEW=false")
-        Domain(name="Default").update({'enabled': True})
-        if rc != 0:
-            logger.exception("Could not re-improt back the Default domain!: `{}`".format(stdout))
-
-
 @pytest.yield_fixture(scope="function")
-def rake(ssh_client, back_up_default_domain):
+def rake(ssh_client):
```

Comment:
I did remove the Default domain saving here in accordance with what we have been doing recently.